configs/e46.h

```diff
@@ -0,0 +1,76 @@
+// Customizations for BMW e46
```

Comment:
Please add
`//#include "configs/e46.h"`
to `config.h` between lines 22 and 23.